doc/source/tools/inlet_profile_generator/inlet_profile_generator.rst

```diff
@@ -0,0 +1,49 @@
+########################
+Inlet profile generator
+########################
+
+This tool is a simple Python script that generates a polynomial function to use as inlet boundary condition for the flow.
+
+The output of the script is a polynomial function that can be used in the ``Function expression`` parameter of a ``inlet`` boundary condition for the velocity.
+
+It can be found in the ``/contrib/utilities/inlet_profile_generator`` folder of the Lethe repository.
+
+To run the script you just need to have `SymPy <https://www.sympy.org/en/index.html>`_ installed in your Python environment. You can install it via pip:
+
+.. code-block:: bash
+    :class: copy-button
+
+    pip install sympy
+
+The script has the following arguments:
+
+* ``--polynomial-degree``: Degree of the polynomial to be generated. Default is ``2```. To prevent negative flow velocities, we always use even degrees.
+* ``--shape``: Shape of the inlet. Supported shapes are ``rectangle`` and ``circle``. Default is rectangle.
+* ``--flow-rate``: Volumetric flow rate to be imposed at the inlet. Default is ``1```.
+* ``--center``: Coordinates of the center of the inlet. Default is ``0. 0.``.
+* ``--length``: Length associated to the inlet shape. If the shape is a rectangle, use ``a,b``, where ``a`` is the length in the x direction and ``b`` is the length in the y direction. If the shape is a circle, give the diameter of. Default is ``1. 1.`` for rectangle and ``1.`` for the circle.
+* ``show``: Used for plotting the generated profile obtained by the script. If ``True`` plots of velocity as a function of ``x`` at center ``y``` and velocity as a function of ``y``` at center ``x`` are shown. Default is ``False``.
+
+Running the script is as simple as:
+
+.. code-block:: bash
+    :class: copy-button
+
+    python3 inlet_profile_generator.py --polynomial-degree 4 --shape circle --flow-rate 2 --center 0. 0. --length 1. --show True
```

Comment:
```suggestion
    python3 inlet_profile_generator.py --polynomial-degree 4 --shape circle --flow-rate 2 --center 0. 0. --length 1. --show 
```

After my comment this would be the new command